blog/_posts/2015-11-20-release-notes-4.3-RC1.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: blog
+title: Scala IDE 4.3-RC1 is out!
+author: Scala IDE team
+disqus: false
+---
+
+We are happy to announce a new Scala IDE release that contains beside of some bug fixes, a new Outline View and a so called Statistics Tracker.
+
+## New and Noteworthy
+
+### New Outline View
+
+The old Outline View, which shows you the structure of the Scala file that is shown in the active editor, got a replacement by a new implementation. The old implementation was provided by the Java IDE of Eclipse and therefore was never a good fit for Scala code. The Scala IDE translated its internal information into a format that could be understood by the Java IDE but this conversion was not without its drawbacks. Since not all syntactic valid Scala elements have a representation in Java, some elements were shown wrong or not at all. The new implementation is a complete rewrite in Scala for Scala code and therefore is a huge improvement over what was available before.
+
+The new Outline looks like this:
+
+<img src="{{site.url}}/resources/images/new-outline.png" alt="New Outline View" />
+
+Note, that the Outline only shows syntactic information. It doesn't show anything that you can't already see in the source code. This means that for example no return type for a method is shown, when there is none in the sources. In case you want more semantic information about your currently opened Scala file, you should have a look to the [scala-search][ss] project, which can be installed through the update-site.
+
+A further note is that all import declarations are collapsed by default. If you want to change this behavior, navigate to `Preferences > Scala > Editor` and disable the behavior in the Outline section.
+
+### Statistics Tracker
+
+Another new feature is the Statistics Tracker. It tracks all the features you use in the Scala IDE and shows you a statistic in the preferences. It can be found in `Preferences > Scala > Statistics` and looks like this:
+
+<img src="{{site.url}}/resources/images/statistics.png" alt="Statistics Tracker" style="width: 100%;" />
+
+The storage location for the statistics tracker can be changed in `Preferences > Scala` to any location you like. Note, that it is possible to share the storage location between multiple instances of Scala IDE running at the same time. In case you do not wish that any data is shared, you need to configure multiple storage locations.
+
+<!--
```

Comment:
This section is disabled for now because I can't get the fix up and running in a test of mine.